lib/mayaUsd/render/MaterialXGenOgsXml/GlslFragmentGenerator.cpp

```diff
@@ -82,8 +82,10 @@ void fixupVertexDataInstance(ShaderStage& stage)
 
     static const std::string primvarParamSource
         = "vec([23]) [$](" + d(HW::T_IN_GEOMPROP) + "_[A-Za-z0-9_]+)";
-
-    static const std::regex primvarParamRegex(primvarParamSource.c_str());
+    static const std::regex  primvarParamRegex(primvarParamSource.c_str());
+    static const std::string texcoordParamSource
+        = "vec([23]) [$](" + d(HW::T_TEXCOORD) + "_[0-9]+)";
+    static const std::regex texcoordParamRegex(texcoordParamSource.c_str());
```

Comment:
As you can see above, we already had code that cleaned up shader stream variable names that were generated for "geompropvalue" nodes. We need to do the same for variable names generated for the "texcoor"nodes used in LookdevX.